tools/feature_matrix.py

```diff
@@ -56,7 +56,7 @@ class Feature(IntEnum):
   Feature.BULK_MEMORY: {
     'chrome': 75,
     'firefox': 79,
-    'safari': 150000,
+    'safari': 140100, # TODO: Reset this to 150000 when we update the default.
```

Comment:
In discussion, we landed on enabling the 3 Safari 15 features separately rather than all at once. In order to do this, I thought of temporarily reducing the required Safari version for each feature to the version that we currently target (14.1). This has the effect of enabling the feature by default without enabling the other 2 safari 15 features by default. Then when all 3 land, we fix them again. WDYT?

Reply:
SGTM